src/wash/common/WithFonts.tsx

```diff
@@ -1,17 +1,8 @@
 'use client';
 
-import React from 'react';
-
 import type { ReactElement, ReactNode } from 'react';
 import { inter } from 'src/fonts/fonts';
-import { Titan_One } from 'next/font/google';
-
-export const titanOne = Titan_One({
-  weight: ['400'],
-  subsets: ['latin'],
-  display: 'swap',
-  variable: '--titan-one-font',
-});
```

Comment:
This part also needs to be used on a server component to render the wash-progress info box properly. In order to avoid the "use client" I seperated this part andI adjusted imports across other files.